src/components/05_pages/NodeForm/index.js

```diff
@@ -147,6 +159,29 @@ class NodeForm extends React.Component {
     schema.properties.data.properties.attributes.properties[fieldName] ||
     schema.properties.data.properties.relationships.properties[fieldName];
 
+  resolveRequiredFiles = () => {
```

Comment:
Files -> Fields

Reply:
Also seem the method name must be something like resolveMissingFields not resolveRequiredFields